message tab_ShieldTimeAttrTable 
{
	optional string tname = 1;
	repeated tab_ShieldTimeAttr tlist = 2;
}

message tab_ShieldTimeAttr
{
	optional int32 id = 1;
	optional string Comment = 2;
	optional int32 InGameName = 3;
	optional int32 Cost = 4;
	optional int32 CoinsMulti = 5;
}